index.d.ts

```diff
@@ -0,0 +1,29 @@
+interface UrlBuilderOptions {
+  allowUpscale: boolean,
+  backgroundColor: string,
+  borderRadius: string,
```

Comment:
```suggestion
  borderRadius: number,
```

Reply:
Fixed in a1d5154eff961fc5d34a36ba7c9c06c0979ff2aa.